pdc_client/__init__.py

```diff
@@ -305,9 +329,24 @@ def __call__(self, *args, **kwargs):
     def __getattr__(self, name):
         return _BeanBagWrapper(self.client.__getattr__(name), self.page_size)
 
+    def __delattr__(self, name):
+        return self.client.__delattr__(name)
+
     def __getitem__(self, *args, **kwargs):
         return _BeanBagWrapper(self.client.__getitem__(*args, **kwargs), self.page_size)
 
+    def __setitem__(self, name, value):
+        return self.client.__setitem__(name, value)
+
+    def __delitem__(self, name):
+        return self.client.__delitem__(name)
+
+    def __iadd__(self, value):
+        return self.client.__iadd__(value)
+
+    def __eq__(self, other):
+        return self.client == other.client
+
```

Comment:
Do we need to add this eq function? Beanbag codes don't init this function.
There are also a str and path function here, should think about if we need re-write them.

BTW: This wrapper class makes us wrapper all the functions.

Reply:
Nice catch. I've added `__str__()`.

Method for equality check is needed - there is [an example for it in BeanBag API docs](http://beanbag.readthedocs.io/en/latest/v1.html).

Reply:
ok, cool